pkg/engine/vulnerability_builder.go

```diff
@@ -25,6 +25,15 @@ const (
 	valuePartsLength = 2
 )
 
+func getStringFromMap(vulnParam, defaultParam string, vOjb map[string]interface{}, logWithFields *zerolog.Logger) string {
```

Comment:
NIce. Should we add a test just for this function?

Reply:
I don't think it is necessary right now, since it is only used on `DefaultVulnerabilityBuilder` and it is already tested...